frontend/e2e/src/pages/configStep.test.ts

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from '@playwright/test'
+
+const PROJECT_NAME_LABEL = 'Project Name *'
+const REGULAR_CALENDAR = 'Regular Calendar(Weekend Considered)'
+const CHINA_CALENDAR = 'Calendar with Chinese Holiday'
+const COLLECTION_DATA = 'Collection Data'
+const FORM_LABEL = 'From *'
+const CHOOSE_DATA = 'Choose date'
+const TO_LABEL = 'To *'
+const ERROR_DATE = '02/20'
+const REQUIRED_DATA = 'Required Data'
+const VELOCITY = 'Velocity'
+const CLASSIFICATION = 'Classification'
+
+const clickCreateNewProjectButton = async (page) => {
```

Comment:
PLZ move some methods like `select` and `fill fields` into a file to share with other steps or flow, you can learn PageObject Model